src/main/java/org/opensearch/flowframework/workflow/CreateIngestPipelineStep.java

```diff
@@ -127,7 +127,7 @@ public CompletableFuture<WorkflowData> execute(List<WorkflowData> data) {
                 logger.info("Created ingest pipeline : " + putPipelineRequest.getId());
 
                 // PutPipelineRequest returns only an AcknowledgeResponse, returning pipelineId instead
-                createIngestPipelineFuture.complete(new WorkflowData(Map.of("pipelineId", putPipelineRequest.getId())));
+                createIngestPipelineFuture.complete(new WorkflowData(Map.of("pipeline_id", putPipelineRequest.getId())));
```

Comment:
Let's try to make nearly all of these strings constants to make everyone's life easier.